array/primenumbers.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import "fmt"
+
+func prime(array []int) {
```

Comment:
func IsPrime(n int) bool{} use this function defination and call this function in main for each array element.